### ISSUE TITLE
Authentication Failing

### ISSUE THREAD

**Issue by chmurph2** [nesquena/sheet_mapper#11]
Hello, I started seeing the following error today when using this gem: 

```
Authentication failed for [my username]: Response code 404 for post https://www.google.com/accounts/ClientLogin: https://developers.google.com/accounts/docs/AuthForInstalledApps
```

https://developers.google.com/accounts/docs/AuthForInstalledApps indicates that ClientLogin is deprecated.  

> Important: ClientLogin has been officially deprecated as of April 20, 2012 and is no longer available as per our deprecation policy. We encourage you to migrate to OAuth 2.0 as soon as possible.

I just wanted to reach out to see if you are aware of this, and have any plans of upgrading this gem to support new methods of authentication (i.e. OAuth 2.0). Regardless, thanks for creating this gem, we've found it useful.

**Comment by nesquena** [comment]
Yeah when they removed the simple login method, this gem broke. To fix this requires switching authentication to OAuth 2. Unfortunately, I likely won't be making the change myself anytime soon  although I will miss this library if it never gets updated. I would of course love to merge in any pull requests and release a new version but I don't expect anyone to do that. Thanks for raising this issue.

**Comment by chmurph2** [comment]
Understood, thanks for the reply.  :beers:
